ang/civicase/DashboardTab.html

```diff
@@ -1,5 +1,194 @@
 <div class="civicase__dashboard__tab">
-  <div class="civicase__dashboard__tab__top"></div>
+  <div class="civicase__dashboard__tab__top">
+    <div class="panel panel-secondary civicase__dashboard__overview">
```

Comment:
don't mention "dashboard" in the class name, call it something like `civicase__case-overview`